### PR TITLE
Update to OIE overview page for PATEAM

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** Update to OIE overview page for PATEAM
- **Is this PR related to a Monolith release?** No

Netlify link: https://66283d50ddf59b2c8a278775--reverent-murdock-829d24.netlify.app/docs/guides/oie-upgrade-overview/main/

### Resolves:

* [OKTA-704690](https://oktainc.atlassian.net/browse/OKTA-704690)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md](https://github.com/okta/okta-developer-docs/blob/d90be6f52febce6f490dbb1ccb3792b446fbfb6a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-upgrade-overview%2Fmain%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c66d7eb9-7556-4616-acee-cdfc48b0060e) | 53  | 43  | 78  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.af5f856d-f284-43cd-901a-d97d15fd47cb?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md](https://github.com/okta/okta-developer-docs/blob/6797b5e066adcb0530dd64aaa9bc1124c76d3e39/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-upgrade-overview%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3f5311c3-124b-41c7-adb3-88ac197a5143) | 51  | 69  | 78  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.056a9a23-cf8a-48ab-87f3-7ef624aff4cd?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by susanharper-okta** [comment]
approved by Dan in Slack

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md [hunk 1]
```diff
@@ -4,29 +4,45 @@ meta:
...
+### Helpful upgrade links
+
+* [Prepare your customizations for upgrade](https://help.okta.com/okta_help.htm?type=oie&id=ext-custom-sign-in-page).
+* [Identify how Okta is implemented](https://www.youtube.com/watch?v=gUqZUSeL_oM&list=PLIid085fSVduvUaN-gBdN1cudndqR9IH8&index=3).
+* [Review the comprehensive list of Identity Engine feature changes](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-features) to identify any potential impact to your upgrade process.
 
 ## Org upgrade process
 
-Your account team will help you to assess your org and schedule the upgrade. If there are any significant remediations required (for example converting and migrating objects such as policies, orgs, and users), they will let you know what services are needed. If there are no remediations required, then the upgrade is simple.
+Access the **OIE Upgrade Hub** in the Admin Console and review any outstanding tasks. If there are any remediation tasks required, those remediation guides appear in the **OIE Upgrade Hub**. If there are no remediations required, then your upgrade is more simple.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md:34]:
```suggestion
Access the **OIE Upgrade Hub** in the Admin Console and review any outstanding tasks. If there are any remediation tasks required, those remediation guides appear in the **OIE Upgrade Hub**. If there are no remediations required, then your upgrade is easier.
```

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md [hunk 2]
```diff
@@ -4,29 +4,45 @@ meta:
...
 
-> **Note**: For more detailed org upgrade instructions, see [Upgrade from Classic Engine](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-upgrade-eligibility).
+For detailed org upgrade instructions, see the [Self-service upgrade process](https://help.okta.com/okta_help.htm?type=oie&id=ext-post-upgrade-validation-tests)
 
-Bear in mind that:
+Keep in mind the following considerations:
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md:38]:
```suggestion
Consider the following:
```

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md [hunk 3]
```diff
@@ -4,29 +4,45 @@ meta:
...
 
-* After upgrading to Okta Identity Engine, your auth experience may continue to mimic Classic Engine, but your admin experience changes and all of the Okta objects from Classic Engine are now compatible with Identity Engine.
-* The existing Okta-hosted Sign-In Widget works as-is after you upgrade your org.
+* After upgrading to Okta Identity Engine, your auth experience may continue to mimic Classic Engine. However, your admin experience changes and all Okta objects from Classic Engine are now compatible with Identity Engine.
+* After you upgrade, the existing Okta-hosted Sign-In Widget works as-is.
 
-> **Note:** See [Upgrade from Classic Engine](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-upgrade-eligibility) for feature comparison and considerations before and after you upgrade.
+The rest of the upgrade process is defined in the next section. Which steps that you take are related to your deployment model. Okta has carefully considered how you can break up the upgrade steps to ensure that you maintain the best user experience across your applications.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md:43]:
```suggestion
The rest of the upgrade process is defined in the next section. The steps that you take are related to your deployment model. Okta has carefully considered how you can break up the upgrade steps to ensure that you maintain the best user experience across your applications.
```

Comment (grahamsmith-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md:43]: don't need which and that in the same sentence

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md [hunk 4]
```diff
@@ -39,32 +55,44 @@ For a more detailed look at the upgrade steps, see the [Plan embedded auth appli
...
-    **Are you moving to an architecture that leverages an Okta SDK?** More detailed information coming soon. <!-- See [Move away from using Classic Authentication APIs to using the appropriate SDK](/docs/guides/) for details on this task. -->
+    * Add the appropriate Identity Engine SDK to your application code. See [Add the latest Auth SDKs to your applications](/docs/guides/oie-upgrade-add-sdk-to-your-app/nodejs/main/) for detailed steps by language.
 
-3. Test your user experience.
+3. Test all of your user experiences.
 
     * **Authentication:** Make sure that your users can sign in and sign out for the workflows that you support.
 
-    * **Self-Service Password Recovery:** Make sure that users can recover their factors with no blockers.
+    * **Self-service & Admin-initiated Password Recovery:** Make sure that users can recover their factors with no impact to the user experience.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md:86]:
```suggestion
    * **Self-service and admin-initiated password recovery:** Make sure that users can recover their factors with no impact to the user experience.
```

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md [hunk 5]
```diff
@@ -39,32 +55,44 @@ For a more detailed look at the upgrade steps, see the [Plan embedded auth appli
...
+
+    * **New Account Activation:** Make sure that the account onboarding experience works as expected end-to-end to ensure that users have a smooth start.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md:88]:
```suggestion
    * **New account activation:** Make sure that the account onboarding experience works as expected end-to-end to ensure that users have a smooth start.
```

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md [hunk 6]
```diff
@@ -39,32 +55,44 @@ For a more detailed look at the upgrade steps, see the [Plan embedded auth appli
...
+
+    * **Self-service registration:** Make sure that users can sign up (including Factor enrollment if you support that) and that there are no interruptions during that process.
+
+    * **Session Management:** If you’re using the Sessions API, some [methods for the Sessions API aren't supported in Identity Engine](https://support.okta.com/help/s/article/v1sessionssessionid-API?language=en_US).
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md:92]:
```suggestion
    * **Session management:** If you’re using the Sessions API, some [methods for the Sessions API aren't supported in Identity Engine](https://support.okta.com/help/s/article/v1sessionssessionid-API?language=en_US).
```

Comment (grahamsmith-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md:92]: I changed each of these to sentence case but if they're actual feature names I'm fine changing them back to capital.

**Review by grahamsmith-okta — CHANGES_REQUESTED** [review]
just a few comments

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md](https://github.com/okta/okta-developer-docs/blob/f4d4e67dec8ae9f91136d0c77b2c1aade9f96b75/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-upgrade-overview%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7460fa58-464d-4fed-83b6-2a9c7b897121) | 51  | 69  | 78  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.0532d24f-1ea6-4f97-8d1b-6cb932fe6723?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by grahamsmith-okta — APPROVED** [review]

**Pull request merged**